src/software/jetson_nano/services/motor.cpp

```diff
@@ -386,21 +386,41 @@ TbotsProto::MotorStatus MotorService::poll(const TbotsProto::MotorControl& motor
     motor_status.mutable_angular_velocity()->set_radians_per_second(
         current_euclidean_velocity[2]);
 
-    WheelSpace_t target_wheel_velocities = {0.0, 0.0, 0.0, 0.0};
     int target_dribbler_rpm =
         motor.drive_control_case() ==
                 TbotsProto::MotorControl::DriveControlCase::DRIVE_CONTROL_NOT_SET
             ? 0
             : motor.dribbler_speed_rpm();
-    static int ramp_rpm = 0;
+    ramp_rpm = 0;
 
-    EuclideanSpace_t prev_euclidean_wheel_velocity =
-        euclidean_to_four_wheel.getEuclideanVelocity(prev_wheel_velocities);
+    WheelSpace_t target_wheel_velocities = WheelSpace_t::Zero();
+
+    if (motor.has_direct_per_wheel_control())
+    {
+        TbotsProto::MotorControl_DirectPerWheelControl direct_per_wheel =
+            motor.direct_per_wheel_control();
+        target_wheel_velocities = {
+            direct_per_wheel.front_left_wheel_velocity(),
+            direct_per_wheel.back_left_wheel_velocity(),
+            direct_per_wheel.front_right_wheel_velocity(),
+            direct_per_wheel.back_right_wheel_velocity(),
+        };
```

Comment:
The order of the values in this matrix incorrect. Here's how it was defined previously:
https://github.com/UBC-Thunderbots/Software/blob/caca93e56f3b4780a01c20fda391f2f995695d4a/src/software/jetson_nano/services/motor.cpp#L400-L404
The ordering is important for the conversion:
https://github.com/UBC-Thunderbots/Software/blob/caca93e56f3b4780a01c20fda391f2f995695d4a/src/software/physics/euclidean_to_wheel.h#L17-L22